pkg/webui/console/views/application-api-key-edit/index.js

```diff
@@ -37,12 +37,14 @@ import {
   applicationKeyErrorSelector,
   applicationKeyFetchingSelector,
 } from '../../store/selectors/application'
+import { selectSelectedApplicationId } from '../../store/selectors/applications'
 
 import api from '../../api'
 
 @connect(function (state, props) {
-  const { appId, apiKeyId } = props.match.params
+  const { apiKeyId } = props.match.params
 
+  const appId = selectSelectedApplicationId(state)
   const keyFetching = applicationKeyFetchingSelector(state)
   const rightsFetching = applicationRightsFetchingSelector(state, props)
```

Comment:
```suggestion
   const rightsFetching = applicationRightsFetchingSelector(state)
```
Unrelated, but this does not consume the props, right?